### ISSUE TITLE
Package and tooling upgrades 

### ISSUE THREAD

**Issue by graforlock** [SodiumFRP/sodium-typescript#16]
- [x] Update Typescript version to preferably 2.0 or higher (2.0.8 is the latest stable).
- [x] NPM package updated to a version 0.9.1 which will include the new API changes and Typescript upgrade.
- [x] Same as above but for bower.

**Comment by graforlock** [comment]
I imagine I have to be logged as ```sodiumjs``` on NPM to publish that

**Comment by the-real-blackh** [comment]
@rixrix Can you give me the authentication credentials necessary to publish Sodium to npm?

**Comment by rixrix** [comment]
hello @graforlock / @the-real-blackh my apology been slacking a bit, my email notifications is failing on my site not until my username was mentioned by Stephen

anyhow,  re: npmjs - I could help out with the publishing updates to npmjs and bower. As for account access,  this is linked to my personal account so either we'll have to transfer ownership or create an org account, let me know for the meantime I'll go ahead with publishing a new npm version. thanks

**Comment by rixrix** [comment]
@graforlock thank for your contribution, your latest changes has been published to npm registry and bower - bower requires a new tag from this repo and so I created a new release tag, feel free to update the release notes should I missed some details -- fyi @the-real-blackh

**Comment by graforlock** [comment]
Thanks!

**Comment by the-real-blackh** [comment]
@rixrix Much appreciated! Could you create an org account? Any of us could get hit by a bus.

**Comment by rixrix** [comment]
re: Any of us could get hit by a bus
errr hope not! 🙏 

npm's org account is bit pricey though so we'll leave it as second option. The doable option would be is hooking up to our CI - TravisCI. We'll have to have a pipeline for our releases so we can tell TravisCI on when and when not publish to npm, bower, etc.

So I'm thinking of the following for our releases pipeline:

* branch `master` : TravisCI does the following: runs unit tests, linting, etc, publish to npm as `next` tag eg: `npm install sodiumjs@next`
* tag `0.9.2`, `0.9.3`, `1.0.0` - ~ : the same with master branch activity but do production release. `npm install sodiumjs@0.9.2` or `npm install sodiumjs`

I'll start working on this and there's also a separate ticket #19 of which I'll add more details in there. comments/suggestions are always welcome.

**Comment by the-real-blackh** [comment]
Thank you. I don't know how much trouble that is. Another option might just be to set up a personal account for Sodium and share the password between several people.

**Comment by rixrix** [comment]
@the-real-blackh yep let's do a Sodium account
